### PR TITLE
Few keywords are mistyped

### PR DESCRIPTION
**Pull request by abhishekyana**
Few args are mistyped please check.

### CODE REVIEW

**Comment by lucidrains** [comment]
Great catch! If you change your pull request so only the keywords for argmax and logsumexp are modified, I'll accept it!

File: reformer_pytorch/reformer_pytorch.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def hash_vectors(self, n_buckets, vecs):
 
         if self._rehash_each_round:
             rotated_vecs = torch.cat([rotated_vecs, -rotated_vecs], dim=-1)
-            buckets = torch.argmax(rotated_vecs, axis=-1)
+            buckets = torch.argmax(rotated_vecs, dim=-1)
```
Comment (lucidrains) [inline reformer_pytorch/reformer_pytorch.py]: Interesting, it works even with `axis`

**Comment by abhishekyana** [comment]
Thanks!! Did it! Please check.

File: reformer_pytorch/reformer_pytorch.py (hunk 1 above)
Comment (abhishekyana) [reply reformer_pytorch/reformer_pytorch.py]: Okay! axis raised an error for me. PyTorch version difference might be the issue. So, adding specific (torch==1.4.0) in requirements.txt will help in 'functional.glu' line  as well.
